data_dictionary.md

```diff
@@ -32,16 +33,16 @@ The Activity Stream addon sends two distinct types of pings to the backend (HTTP
 
 ```json
 {
-  
+  "client_id": "374dc4d8-0cb2-4ac5-a3cf-c5a9bc3c602e",
+  "tab_id": "1-3",
+  "action": "activity_stream_event",
+  "addon_version": "1.0.0",
   "source": "top sites, or bookamrks, or...",
```

Comment:
While we're in this file, would you mind fixing the "bookamrks" typo. Sorry, I should have caught that earlier.